javascript/javascript1/week4/voiceAssistant.js

```diff
@@ -0,0 +1,52 @@
+let nameHolder= "";
+const toDoList = [];
+const monthNames = ["January", "February", "March", "April", "May", "June",
+  "July", "August", "September", "October", "November", "December"
+];
+function getReply(command) {
+ 
+  // later down fields will be checked if includes some arithmetic statement. 
+  let fields = command.split('is');
+  
+    if (command === 'Hello my name is Benjamin' && nameHolder==="") {
+
+            const namePosition = command.search("is")+3;
+            nameHolder = command.substr(namePosition, command.length-1);
+            return "Nice to meet you Benjamin";
+  }
+  else if (command === 'Hello my name is Benjamin'){
+             return 'You already introduced to me!'
+  }
+  else if (command ==='What is my name') {
+      let xx= nameHolder ? nameHolder :`You haven't introduced to me!`;
+      return xx;
+        }
+  else if (command === 'Add fishing to my todo'){
+      toDoList.push('fishing') ;
+      return "fishing added to your todo";
```

Comment:
what you'd want to `return` here is actually an updated `toDoList`. And "fishing added to your todo" should be logged to the console.

Reply:
For example: 
```  else if (command === 'Add fishing to my todo') {
    toDoList.push('fishing');
    console.log("fishing added to your todo: ", toDoList);
    return toDoList;
```